.github/workflows/push.yml

```
@@ -172,15 +172,25 @@ jobs:
           -total-entries-written 300000 \
           -total-entries-read 10000
 
+  go-version:
+    name: Extract Go version
+    runs-on: ubuntu-latest
+    outputs:
+      go-version: ${{ steps.extraction.outputs.go_version }}
+    steps:
+      - id: extraction
+        run: echo "go_version=$GO_VERSION" >> $GITHUB_OUTPUT
+
   performance-tests:
     name: Performance tests
     needs:
       - gosec
       - old-go
+      - go-version
     uses: ./.github/workflows/performance.yml
     secrets: inherit
     with:
-      go-version: ${{ env.GO_VERSION }}
+      go-version: ${{ needs.go-version.outputs.go-version }}
```

Shouldn't this be:

```suggestion
      go-version: ${{ needs.go-version.outputs.go_version }}
```
?

---

No. The second `go-version` on line 193 refers to the `output` property of the file on line 179, not to the variable on line 182. I know it's a bit confusing, but I wanted to have an underscore on the variable name.